swssconfig/swssplayer.cpp

```diff
@@ -34,7 +37,30 @@ vector<FieldValueTuple> processFieldsValuesTuple(string s)
 	return result;
 }
 
-void processTokens(vector<string> tokens)
+shared_ptr<ProducerStateTable> get_table(unordered_map<string, shared_ptr<ProducerStateTable>>& table_map, string table_name, set<string>  zmq_tables, std::shared_ptr<ZmqClient> zmq_client)
```

Comment:
This method very similar with the get_table method in swssconfig
However because the ProducerStateTable in this method create with RedisPipeline and has different parameter number, so can't create a template method for code reuse.